### DISCUSSION TITLE
Does StaEDI supports NCPDP D.0 B1, B2. B3 standards?

### DISCUSSION THREAD

**Discussion by dineshraghuramaiah** [xlate/staedi#500] [General]
Mike, Does StaEDI supports NCPDP D.0 B1, B2. B3 standards?

**Comment by MikeEdgar** [comment]
Hi @dineshraghuramaiah , NCPDP is not support right now. I've looked in the past for information on the standard and specifications for the formatting, but I have not been able to turn up enough to implement it. If you are aware of documentation/specifications for NCPDP, it would be nice to have it in a future release.

**Reply by dineshraghuramaiah** [reply]
https://datainsight.health/ncpdp/intro/. please check this.

**Comment by MikeEdgar** [comment]
Closing discussion and further comments can be added to #504 for this enhancement.
